Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.10 as builder
+
+RUN mkdir -p /go/src/github.com/src-d/gitbase
+WORKDIR /go/src/github.com/src-d/gitbase
+COPY . .
+
+RUN apt-get update && apt-get install libxml2-dev -y
+RUN go get github.com/golang/dep/...
+RUN dep ensure
+RUN cd vendor/gopkg.in/bblfsh/client-go.v2 && make dependencies
+RUN go install -v github.com/src-d/gitbase/...
+
+FROM ubuntu:16.04
+
+COPY --from=builder /go/bin/gitbase /bin
+RUN mkdir -p /opt/repos
+
+ENV GITBASE_USER=gitbase
+ENV GITBASE_PASSWORD=""
```

Comment:
just because I'm curious - do we rely on some image where `GITBASE_PASSWORD` is exported or we're just using empty password?

Reply:
We're using no password by default and then you can add your own password when you run it using `-e GITBASE_PASSWORD="your password"`